test/module/irohad/torii/CMakeLists.txt

```diff
@@ -37,4 +37,13 @@ target_link_libraries(torii_queries_test
 addtest(query_service_test query_service_test.cpp)
 target_link_libraries(query_service_test
     torii_service
+    server_runner
```

Comment:
Query service test is not changed in this pull request, so these libraries should not be present here.